src/simplify_qdq.cpp

```diff
@@ -38,6 +38,7 @@
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
+namespace {
```

Comment:
were there any symbol conflicts that required adding another namespace ?

Reply:
Not that I know, but this will help prevent ODR issues(which is difficult to debug) in the future. We should use this for all classes define in .cpp files though.